src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -44,35 +45,35 @@ public void testIsFull() {
 
     @Test
     public void testPeekaboo() {
-        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-        final int EMPTY_STACK_VALUE = -1;
+        DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer(new IntArrayStack());
+        final int EMPTY_STACK_VALUE = 0;

```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).